packages/x-data-grid/src/locales/nlNL.ts

```diff
@@ -39,10 +39,10 @@ const nlNLGrid: Partial<GridLocaleText> = {
   toolbarExportExcel: 'Downloaden als Excel-bestand',
 
   // Columns management text
-  // columnsManagementSearchTitle: 'Search',
-  // columnsManagementNoColumns: 'No columns',
-  // columnsManagementShowHideAllText: 'Show/Hide All',
-  // columnsManagementReset: 'Reset',
+  columnsManagementSearchTitle: 'Zoeken',
+  columnsManagementNoColumns: 'Geen kolommen',
+  columnsManagementShowHideAllText: 'Toon/Verberg Alle',
+  columnsManagementReset: 'Reset',
```

Comment:
```suggestion
  // columnsManagementReset: 'Reset',
```

Reply:
This is intentional. You can use this in Dutch. Alternatives would likely be too long for this UI. 
https://woordenlijst.org/zoeken/?q=reset

Reply:
I take it that we happy with the proposed translation. 🤔 
In any case, this label would equal `Reset` in `nl-NL` locale and if someone will see it problematic, they can propose a better alternative. 👍